src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -231,6 +281,178 @@
+      "kind": "library",
+      "version": "@MicrosoftExtensionsFileProvidersAbstractionsPackageVersion@"
+    },
+    "Microsoft.Extensions.FileProviders.Composite": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsFileProvidersCompositePackageVersion@"
+    },
+    "Microsoft.Extensions.FileProviders.Embedded": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsFileProvidersEmbeddedPackageVersion@"
+    },
+    "Microsoft.Extensions.FileProviders.Physical": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsFileProvidersPhysicalPackageVersion@"
+    },
+    "Microsoft.Extensions.FileSystemGlobbing": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsFileSystemGlobbingPackageVersion@"
+    },
+    "Microsoft.Extensions.Hosting": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsHostingPackageVersion@"
+    },
+    "Microsoft.Extensions.Hosting.Abstractions": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsHostingAbstractionsPackageVersion@"
+    },
+    "Microsoft.Extensions.Logging": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingPackageVersion@"
+    },
+    "Microsoft.Extensions.Logging.Abstractions": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingAbstractionsPackageVersion@"
+    },
+    "Microsoft.Extensions.Logging.Configuration": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingConfigurationPackageVersion@"
+    },
+    "Microsoft.Extensions.Logging.Console": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingConsolePackageVersion@"
+    },
+    "Microsoft.Extensions.Logging.Debug": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingDebugPackageVersion@"
+    },
+    "Microsoft.Extensions.Logging.EventLog": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingEventLogPackageVersion@"
+    },
+    "Microsoft.Extensions.Logging.EventSource": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsLoggingEventSourcePackageVersion@"
+    },
+    "Microsoft.Extensions.Options": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsOptionsPackageVersion@"
+    },
+    "Microsoft.Extensions.Options.ConfigurationExtensions": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsOptionsConfigurationExtensionsPackageVersion@"
+    },
+    "Microsoft.Extensions.Primitives": {
+      "kind": "library",
+      "version": "@MicrosoftExtensionsPrimitivesPackageVersion@"
+    },
+    "Microsoft.JSInterop": {
+      "kind": "library",
+      "version": "@MicrosoftJSInteropPackageVersion@"
+    },
+    "Microsoft.Maui.Graphics": {
+      "kind": "library",
+      "version": "@MicrosoftMauiGraphicsVersion@"
+    },
+    "Microsoft.Maui.Graphics.Win2D.WinUI.Desktop": {
+      "kind": "library",
+      "version": "@MicrosoftMauiGraphicsWin2DVersion@"
```

Comment:
This version identifier doesn't seem to match the package name, but I'm not sure what it should be because I can't seem to find the name I would "expect" it to be: `MicrosoftMauiGraphicsWin2DWinUIDesktopPackageVersion`

Reply:
These `Microsoft.Maui.Graphics` packages aren't on Maestro, they have custom property names:

https://github.com/dotnet/maui/blob/5adf0b11f24eec22ff3e9ff0b5943728ce47f33e/src/Workload/Microsoft.NET.Sdk.Maui/Microsoft.NET.Sdk.Maui.csproj#L64-L66

I should add `PackageVersion` on the end of the name in this file, though to be consistent.